UI/Sources/ShoppingCart/Models/PaymentMethodManager.swift

```diff
@@ -187,35 +166,64 @@ extension PaymentMethodItem {
     }
 }
 
-extension Project {
-    public func paymentItems(for shoppingCart: ShoppingCart? = nil) -> [PaymentMethodItem] {
-        let allAppMethods = Set(
-            Snabble.shared.projects
-                .flatMap { $0.paymentMethods }
-                .filter { $0.isAvailable }
-        )
+extension Snabble {
+    public func allAvailablePaymentMethods() -> [RawPaymentMethod] {
+        projects
+            .flatMap(\.paymentMethods)
+            .filter(\.isAvailable)
+    }
+}
 
+extension Project {
+    public var orderedPaymentMethods: [RawPaymentMethod] {
+        let allAppMethods = Snabble.shared.allAvailablePaymentMethods()
```

Comment:
Du bekommst hier alle Payment Methoden über alle Projekte anstatt nur für das Projekt in dem du gerade bist.

Reply:
Die bisherige Implementierung zeigt die Zahlungsmethoden für alle Projekte an.
Die neue PaymentSelectionView erlaubt die Angabe von verfügbar Zahlungsmethoden. Im Shopper gibt es nun `projectPayments`, die über den Parameter availablePayments der View übergeben wird.

Reply:
https://github.com/snabble/snabble-ios-sdk/pull/400/commits/dd59f623d40c5ce0cda7df5d0160f45930b37524

Reply:
Jetzt werden nur die Zahlungsmethoden des Projektes verwendet.
https://github.com/snabble/snabble-ios-sdk/pull/400/commits/fcd3fdff4c7364f6e4474fade0461a3ae94a57b7